moneyCoins.js

```diff
@@ -1,13 +1,22 @@
-function moneyCoins (money) {
-  // your implementation code here
-}
-
-// Drive code
-console.log(moneyCoins(543))
-//output : [ 500, 20, 20, 1, 1, 1 ]
-
-console.log(moneyCoins(7752))
-//output : [ 5000, 2000, 500, 200, 50, 1, 1 ]
-
-console.log(moneyCoins(37454))
-//output : [ 10000, 10000, 10000, 5000, 2000, 200, 200, 50, 1, 1, 1, 1 ]
+function converttoCoin (coin) {
+  var coins = [10000, 5000, 2000, 1000, 500, 200, 100, 50, 20, 10, 1];
+  var result = [];
+      while(coin > 0) {
```

Comment:
ini udah oke kok logic nya.. good job yaa.. :D

Bisa juga disimpelkan kayak gini.. tanpa harus while(coin < 0) diatas nya.

          for (var i = 0; i < coins.length; i++) {
              while (coin >= coins[i]) {
                  result.push(coins[i]);
                  coin -= coins[i];
              }
          }

Reply:
btw, Ini cuma saran,, tidak mengurangi nilai.

Reply:
makasih kak